vllm/core/block_manager.py

```diff
@@ -440,9 +533,10 @@ def access_all_blocks_in_seq(
         seq: Sequence,
         access_time: float,
     ) -> None:
-        block_table = self.block_tables[seq.seq_id]
-        for block in block_table:
-            block.last_accessed = access_time
+        if self.enable_caching:
+            block_table = self.block_tables[seq.seq_id]
+            for block in block_table:
+                block.last_accessed = access_time
```

Comment:
Please add a comment to make the logic more clear for new comers.
```suggestion
        if self.enable_caching:
            # Update the last accessed time of all the blocks accessed in this step
            block_table = self.block_tables[seq.seq_id]
            for block in block_table:
                block.last_accessed = access_time
```